editor/export/editor_export.cpp

```diff
@@ -227,6 +228,7 @@ void EditorExport::load_config() {
 		}
 
 		preset->set_name(config->get_value(section, "name"));
+		preset->set_advanced_options_enabled(config->get_value(section, "advanced_options"));
```

Comment:
This should probably give a default value:

```suggestion
		preset->set_advanced_options_enabled(config->get_value(section, "advanced_options", false));
```

Currently, when I open an existing project, I get an error for each existing export preset:

![Selection_118](https://github.com/godotengine/godot/assets/191561/5a4f9359-d3b3-4aa5-afd3-c6456e7a1316)

Reply:
Fixed!